<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playback</class>
 <widget class="QWidget" name="Playback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>468</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>441</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="page_playback">
    <widget class="PlaybackLayout" name="playbackLayout" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>281</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>7</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="PlaybackBar" name="playbackBar" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>471</width>
       <height>56</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="PicturePlay" name="page_picture"/>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_right">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>109</width>
     <height>448</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="PlaybackMode" name="page_mode"/>
   <widget class="QWidget" name="page_list"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaybackLayout</class>
   <extends>QWidget</extends>
   <header>PlaybackLayout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlaybackBar</class>
   <extends>QWidget</extends>
   <header>PlaybackBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PicturePlay</class>
   <extends>QWidget</extends>
   <header>PicturePlay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlaybackMode</class>
   <extends>QWidget</extends>
   <header>PlaybackMode.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
